src/ContainerCommandLoader.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-cli for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-cli/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-cli/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Laminas\Cli;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\CommandLoader\ContainerCommandLoader as SymfonyContainerCommandLoader;
+
+/**
+ * @internal
+ */
+final class ContainerCommandLoader extends SymfonyContainerCommandLoader
+{
+    public function get(string $name) : Command
+    {
+        $command = parent::get($name);
+        $command->setName($name);
```

Comment:
I like this A LOT.

It means you can specify a default command name, but the _application_ gets to override it via the configuration map.

Nice solution!

Reply:
Yes... unless you change the name in `configure`. And this is another problem, maybe not a big problem :)